cost-model/src/cost_model.rs

```diff
@@ -659,42 +643,6 @@ mod tests {
         );
     }
 
-    #[allow(clippy::field_reassign_with_default)]
-    #[test]
-    fn test_calculate_loaded_accounts_data_size_cost() {
```

Comment:
we're losing this testing, seems like they should be moved to `FeeStructure`'s tests, wdyt?

Reply:
I was expecting FeeStructure has tests to cover these logics ;) yea, will make sure the tests are still covered.

Reply:
34b58b4431e10070cf6dd984f934ae53f159de47 added tests to FeeStructure